File: exercicios/1018/cedulas.java
import java.io.IOException;
import java.util.Scanner;


public class Main {
    
    public static void main(String[] args) throws IOException {
        int n;
        Scanner input = new Scanner(System.in);
        n = input.nextInt();
        System.out.printf("%d nota(s) de R$ 100,00\n", n / 100);
        n = n % 100;
        System.out.printf("%d nota(s) de R$ 50,00\n", n / 50);
        n = n % 50;
        System.out.printf("%d nota(s) de R$ 20,00\n", n / 20);
        n = n % 20;
        System.out.printf("%d nota(s) de R$ 10,00\n", n / 10);
        n = n % 10;
        System.out.printf("%d nota(s) de R$ 5,00\n", n / 5);
        n = n % 5;
        System.out.printf("%d nota(s) de R$ 2,00\n", n / 2);
        n = n % 2;
        System.out.printf("%d nota(s) de R$ 1,00\n", n); 
        input.close();  
    }    
}

File: exercicios/1018/cedulas.kt
import java.util.Scanner


fun main( args: Array<String> ){

    val input = Scanner(System.`in`)
    var g : Int = input.nextInt()

    println ("${g / 100} nota(s) de R$ 100,00")
    g = g % 100
    println ("${g / 50} nota(s) de R$ 50,00")
    g = g % 50
    println ("${g / 20} nota(s) de R$ 20,00")
    g = g % 20
    println ("${g / 10} nota(s) de R$ 10,00")
    g = g % 10
    println ("${g / 5} nota(s) de R$ 5,00")
    g = g % 5
    println ("${g / 2} nota(s) de R$ 2,00")
    g = g % 2
    println ("${g} nota(s) de R$ 1,00")
}